src/commands/admin/prefix.js

```diff
@@ -1,5 +1,6 @@
 const { setServerPrefix, getServerPrefix } = require("../../utils/functions");
 const { ownerId } = require("../../../config.json");
+const { MessageEmbed } = require("discord.js");
```

Comment:
Please remove this line as It's not being used.

Reply:
oh i thought u were gonna use it for modlog

Reply:
well ok then

Reply:
done